image-pipeline/expression-scatter-plots.js

```diff
@@ -23,8 +25,8 @@ const { values } = parseArgs({ args, options })
 
 // Candidates for CLI argument
 // CPU count on Intel i7 is 1/2 of reported, due to hyperthreading
-const numCPUs = os.cpus().length / 2 - 1
-// const numCPUs = 2
+// const numCPUs = os.cpus().length / 2 - 1
```

Comment:
Should there be a comment here about why the numCPUs is just 1 rather than using this commented out logic?

Reply:
Done in 505c492.